src/main/kotlin/inc/evil/d17map/controllers/ReservationController.kt

```diff
@@ -113,7 +116,7 @@ class ReservationController(private val reservationService: ReservationService)
     @GetMapping("/week")
     fun getReservationsForWeek(@RequestParam("monday") monday: LocalDate?): ResponseEntity<Any> {
         if (monday == null) {
-            return ResponseEntity("Query parameter 'monday' must be specified", HttpStatus.BAD_REQUEST)
+            throw MissingParameterException("monday")
```

Comment:
I would change it to sth like "startOfTheWeel" in error message. We know what "monday" means in this situation, but the response code is not so clear in my opinion